Sources/LeafErrorMiddleware/LeafErrorMiddleware.swift

```diff
@@ -19,6 +19,8 @@ public final class LeafErrorMiddleware: Middleware, Service {
                 } else {
                     return try res.encode(for: req)
                 }
+            }.catchFlatMap { error in
+                return try self.handleError(for: req, status: error.localizedDescription.contains("Abort.404") ? .notFound : .forbidden)
```

Comment:
Sorry should have been clearer. On this line, where you're trying to convert the error to a status code, you should cast the error caught to `AbortError` or `Abort` and use the status code property on that instead of defaulting to `.forbidden` or `.notFound`. If the cast fails, then fall back to `internalServerError`. Make sense?

Reply:
(Like https://github.com/vapor/vapor/blob/master/Sources/Vapor/Middleware/ErrorMiddleware.swift#L35)

Reply:
Sorry, now it's good, I think ! Thanks for precisions.